CONTRIBUTING.md

```diff
@@ -3,20 +3,56 @@ Feel free to contribute PRs, Issues, Documentation, or in any other way to this
 
 # Dev Environment Setup
 This application has a development environment that supports Docker. We believe this makes it easier to get started with the application.
+To get started, you will need either Docker [desktop](https://docs.docker.com/desktop/#download-and-install) or Docker [engine](https://docs.docker.com/engine/install) as well as
+Docker Compose which is a tool that is used for defining and running multi-container Docker applications. In our case for running the web application, database and selenium for testing,
 
 ## First time setup
-1. Setup web and db containers. `docker compose up -d`
-1. Run database migrations. `docker compose run --rm web bundle exec rake db:create`
-1. The application should be available at [localhost:3000](localhost:3000)
+1. Setup web and db containers. `docker-compose up -d`
+2. Create database tables. `docker-compose run --rm web bundle exec rake db:create`
+3. Run database migrations. `docker-compose run --rm web bundle exec rake db:migrate`
```

Comment:
`1.` is automatically turned into a numbered list, and then you can add a new item wherever, and it maintains the numbering.

```suggestion
1. Setup web and db containers. `docker compose up -d`
1. Create database tables. `docker compose run --rm web bundle exec rake db:create`
1. Run database migrations. `docker compose run --rm web bundle exec rake db:migrate`
```

Docker has absorbed compose and now you can just run `docker compose` - https://docs.docker.com/compose/cli-command/

Maybe we can make a note that older versions of docker need the `-`?

Reply:
TIL! That's really cool. Yup, we probably should so no one makes the same mistake I was ahahaha. 😂